spec/formatting.md

```diff
@@ -721,6 +725,26 @@ These are divided into the following categories:
     > when * {The value is not one.}
     > ```
 
+  - **Unsupported Expression errors** occur when an expression uses
+    syntax reserved for future standardization,
+    or for private implementation use that is not supported by the current implementation.
```

Comment:
I think we should try to be consistent when talking about private use. Also, remember your formatting. I'd suggest:

```suggestion
or for _annotations_ that use a _private-use_ sequence unsupported by the current implementation.
```

Reply:
Your proposed change does not match our current description of _reserved_, which does not so differentiate annotations reserved for future spec versions compared to those reserved for private implementation use. Furthermore, we do not have a definition of _private-use_ to refer to.

It's clear that the language around _reserved_ could be updated, but that really ought to happen holistically rather than as a part of this PR, which so far is only touching the formatting parts of the spec.